.gitlab-ci.yml

```diff
@@ -89,11 +89,15 @@ trigger:mender-dist-packages:
   stage: trigger
   before_script:
     - apk add --no-cache curl
+    - export MENDER_CLIENT_VERSION=$(git tag | egrep -e '^[0-9]+\.[0-9]+\.[0-9]+$' | sort -V -r | head -n1)
```

Comment:
Not important neither, but for future reference. You don't need `export` here: GitLab just appends `before_script` and `script` and runs it on a shell, so the variables are visible.

Reply:
Thx, noted and removed.